web/src/tippyjs.js

```diff
@@ -292,6 +292,28 @@ export function initialize() {
         },
     });
 
+    delegate("body", {
+        target: [".settings-radio-input-parent.disabled_setting_tooltip"],
```

Comment:
This selector doesn't seem specific to this particular element; I worry this tooltip will apply to other disabled setting tooltips. You need a more precise selector for both this and the next one.

Reply:
I believe there is no function to avoid hardcoding `input[value=...]`. Instead, I used a unique class to target the specific elements.

Reply:
Yeah, adding a new class is the way to go, thanks.